assets/app/view/game/round/auction.rb

```diff
@@ -179,6 +181,33 @@ def render_input(company)
           h(:div, { style: { textAlign: 'center', margin: '1rem' } }, company_actions)
         end
 
+        def render_move_bid_inputs(company, input)
+          return [] unless @current_actions.include?('move_bid')
+
+          moveable_bids = @step.moveable_bids(@current_entity, company)
+          return [] if moveable_bids.empty?
+
+          moveable_bids.map do |old_company, old_bids|
+            old_bids.map do |old_bid|
+              move_bid = lambda do
+                hide!
+                price = input.JS['elm'].JS['value'].to_i
+                process_action(Engine::Action::MoveBid.new(
+                  @current_entity,
+                  company: company,
+                  old_company: old_company,
+                  price: price,
+                  old_price: old_bid.price,
```

Comment:
is it necessary to store the old price? isn't that inferred from the old_company?

what do you think about changing the name to be from_company, to_company, price

Reply:
It is required. A Player can have multiple bids on a single Company, passing the price associated is the key to which of those bids are being moved.

Reply:
Changing it to "from_company" and "to_company" seems good.

Reply:
I made the change from `old_...` to `from_...` but I did not make the change to `to_...` as this allows the bid company and amount to be the same as the normal bid action and thus it greatly simplifies the logic on comparing bid values.